Core/include/Acts/Fitter/KalmanFitter.hpp

```diff
@@ -343,18 +343,19 @@ class KalmanFitter {
 
         // Reset smoothed status of states missed in backward filtering
         if (backwardFiltering) {
-          result.fittedStates.applyBackwards(result.trackTip, [&](auto state) {
-            auto fSurface = &state.referenceSurface();
-            auto surface_it = std::find_if(
-                result.passedAgainSurfaces.begin(),
-                result.passedAgainSurfaces.end(),
-                [=](const Surface* surface) { return surface == fSurface; });
-            if (surface_it == result.passedAgainSurfaces.end()) {
-              // If backward filtering missed this surface, then there is
-              // no smoothed parameter
-              state.data().ismoothed = detail_lt::IndexData::kInvalid;
-            }
-          });
+          result.fittedStates.applyBackwards(
+              result.trackTip, [&](auto trackState) {
```

Comment:
The actual change here is `state` -> `trackState`, right? The rest is formatting?

Reply:
Yes.